index.html

```diff
@@ -1,17 +1,30 @@
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/my_website.css">
     <title>Alejandro Ortiz</title>
 </head>
 
 <body>
-    <p>Hi, I’m @alejandro-ortiz <br>
-        I’m interested in Rocketry, Tennis, Web Development, Electronics <br>
-        I’m currently learning Git and Javascript
-    </p>
+    <header>
+        <h1>Alejandro Ortiz</h1>
+    </header>
+    <main>
+        <p>Hi, I’m Alejandro Ortiz <br> I’m interested in Rocketry, Tennis, Web Development, Electronics, and
+            Scouts<br> I’m currently learning Git and JavaScript
+        </p>
+        <p>❤️My dream job is to work at Apple by doing web development and helping with integrateing ideas.
+        </p>
+        <p>❤️My dream goal is to help as many people understand electronics as I can and understand the components too.
+        </p>
+        <p>My favorite things to use are VSCode, Git, GitHub, MacOS.
```

Comment:
VSCode should be VS Code

Reply:
Yeah I thought it looked a bit funny how I had it